bench/plutus-scripts-bench/plutus-scripts-bench.cabal

```diff
@@ -84,7 +84,8 @@ library
     , text
 
 executable gen-plutus
-  import:               project-config
+  import:               common-definitions
+                      , project-config
```

Comment:
I think we might want to consider this as a separate addition? @mgmeier @NadiaYvette 
The rest of this PR are part of #5160